src/locale/locales/de/messages.po

```diff
@@ -1571,20 +1320,20 @@ msgstr "Kopieren"
 
 #: src/view/com/modals/ChangeHandle.tsx:474
 msgid "Copy {0}"
-msgstr "{} kopieren"
+msgstr "Kopiere {0}"
```

Comment:
```suggestion
msgstr "{} kopieren"
```

no change required

Reply:
This was one of the corrections from @PythooonUser, but you think it's better the other way around?

Reply:
here's the context of `src/view/com/modals/ChangeHandle.tsx:474`:
```ts
Copy {isDNSForm ? _(msg`Domain Value`) : _(msg`File Contents`)}
```

I'd say `Dateiinhalt kopieren` sounds better than `Kopiere Dateiinhalt`

Reply:
I'll just fix the missing variable then
```suggestion
msgstr "{0} kopieren"
```